build.gradle

```diff
@@ -78,7 +87,7 @@ dependencies {
 
   compile group: 'org.hibernate', name: 'hibernate-validator', version: '5.4.1.Final'
 
-  compile group: 'uk.gov.hmcts.reform', name: 'java-logging-spring', version: '1.4.0'
+  compile group: 'uk.gov.hmcts.reform', name: 'java-logging-spring', version: '2.2.1'
```

Comment:
😯

Reply:
1.4.0 didn't have the base exception referenced by one of your classes. That's why I had to bump it.

Reply:
ah. then remove the base class - let's not introduce that nonsense for now

Reply:
What about those alert levels then? Can we live without them?

Reply:
in my opinion yes - we can. draft store as of this state did not have them anyway. just scope out of this PR the logging library bump and replace base class with runtime exception. then 👍 👍 

if application insights has to come in, then we have no choice but to scan all those exceptions and stick in alert levels

Reply:
Alright then, done